addons/website_blog/tests/test_website_blog_flow.py

```diff
@@ -108,3 +108,10 @@ def test_blog_comment(self):
 
         self.assertFalse(self.env['mail.message'].sudo().search(
             [('model', '=', 'blog.post'), ('attachment_ids', 'in', second_attachment.ids)]))
+
+    def test_website_blog_teaser_content(self):
+        """ Make sure that the content of the post is correctly rendered in proper plain text. """
+
+        self.test_blog_post.write({'content': "<h2>Test Content</h2>"})
+
+        self.assertEqual(self.test_blog_post.teaser, " Test Content ...")
```

Comment:
Thank you for the test.
For tracability: we decided that it was not worth introducing code to not add the `...` when the description was < 200 char. While it doesn't makes sense to add those `...`, a blog post is not supposed to ever have less than 200 char, so it actually makes sense.